### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Fixed multiple broken anchor links and update some of the files name that were changed.

## Description

page with multiple outdated content like broken anchor links, some of the files name were changed. Fixed anchor link with filename.

## Affected Page

https://magento.github.io/pwa-studio/venia-pwa-concept/project-structure/

### Resolved issues:
1. [x] resolves magento/pwa-studio#2951: Update index.md

### CODE REVIEW

File: pwa-devdocs/src/venia-pwa-concept/project-structure/index.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ The `venia-static` directory contains the `favicon.ico` icon file, `icons` folde
...
 It sets the ApolloProvider, the ReduxStore, and the Router configuration and App Component.
 
-### `src/store.js`
+### [`src/store.js`][]
 
 The `src/store.js` file is responsible for creating a Redux store.
 It also combines Redux reducers and middlewares.
 
-### `src/sw.js`
+### [`src/registerSW.js`][]
```
Comment (revanth0212) [inline pwa-devdocs/src/venia-pwa-concept/project-structure/index.md:53]: I don't think this is accurate. `registerSw.js` contains logic to register a SW on the app. `sw.js` contains the main Service Worker logic. `sw.js` is the first file from where the webpack bundling of the SW starts.

**Review by revanth0212 — CHANGES_REQUESTED** [review]
Thank you, @RakeshJesadiya for the PR. I have made a small comment about the SW file. Please check.

**Comment by sirugh** [comment]
@magento create issue from PR

**Comment by sirugh** [comment]
Closing for lack of movement.

**Pull request closed without merging**